### DISCUSSION TITLE
zluda vs directML - Gap performance on 5700xt

### DISCUSSION THREAD

**Discussion by zewho** [lshqqytiger/stable-diffusion-webui-directml#493] [Q&A]
Hi,

After a git pull yesterday, with my 5700xt

Using Zluda to generate a 512x512 image gives me 10 to 18s /it
Switching back to directML, i've got an acceptable 1.20 it/s

I tried to adjust my args but still got the same range of results.

As i never used Zluda before, i wanted to know if it is normal to have worst results with zudla than with directML
Maybe there is something outside A1111 to adjust ?

**Comment by lshqqytiger** [comment]
ZLUDA and DirectML are not something easily 'switch'able.
They use different torch build, so you should rebuild venv. (or at least reinstall torch and torchvision)
In addition, ZLUDA has its own requirements rather than DirectML.

**Reply by lshqqytiger** [reply]
You can check [this wiki page](https://github.com/vladmandic/automatic/wiki/ZLUDA) for guide.
It is for SD.Next, but SD.Next and this fork share same installer code of ZLUDA.

**Comment by zewho** [comment]
i'll try that,thanks a lot
